isb_lib/geome_adapter.py

```diff
@@ -150,13 +253,11 @@ def _loadEntries(self):
         for p in self._project_ids:
             # return the next set of identifiers within a project
             if not p["loaded"]:
-                for record in self.recordsInProject(p["project_id"], self._record_type):
+                for (record, modified_date) in self.recordsInProject(p["project_id"], self._record_type):
                     # record identifier
                     rid = record.get("bcid", None)
-                    # record timestamp
-                    t_collected = geomeEventRecordTimestamp(record)
                     if not rid is None:
-                        p["identifiers"].append((rid, t_collected))
+                        p["identifiers"].append((rid, modified_date))
```

Comment:
This is the change to store the modified date of the expedition rather than the collection time (what we had previously).